lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/DBOTeamDAOImpl.java

```diff
@@ -505,15 +518,45 @@ public void delete(String id) {
 	}
 
 	@Override
-	public List<TeamMember> getMembersInRange(String teamId, long limit, long offset)
+	public List<TeamMember> getMembersInRange(String teamId, TeamMemberTypeFilterOptions memberType, long limit, long offset)
 			throws DatastoreException {
 		MapSqlParameterSource param = new MapSqlParameterSource();	
 		param.addValue(COL_GROUP_MEMBERS_GROUP_ID, teamId);
 		param.addValue(OFFSET_PARAM_NAME, offset);
 		if (limit<=0) throw new IllegalArgumentException("'limit' param must be greater than zero.");
-		param.addValue(LIMIT_PARAM_NAME, limit);	
-		List<TeamMember> teamMembers = namedJdbcTemplate.query(SELECT_MEMBERS_OF_TEAM_PAGINATED, param, TEAM_MEMBER_ROW_MAPPER);
-		setAdminStatus(teamMembers);
+		param.addValue(LIMIT_PARAM_NAME, limit);
```

Comment:
always get list of admin ids, refactor setAdminStatus() to to take that list as a parameter